src/ui/home/components/qrcodeScanner.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { View, Text, TextInput, StyleSheet, Linking, TouchableOpacity, Platform, BackHandler } from 'react-native'
```

Comment:
Are Linking, View, or TextInput needed here?

Reply:
Good point. I added a check in tsconfig to show a warning on unused imports.